cli/src/main/scala/co/topl/brambl/cli/controllers/WalletController.scala

```diff
@@ -35,6 +42,84 @@ class WalletController[F[_]: Sync](
     genusQueryAlgebra: dataApi.GenusQueryAlgebra[F]
 ) {
 
+  def addSecret(
+      secretTxt: String,
+      digest: DigestType
+  ): F[Either[String, String]] = {
+    import co.topl.crypto.hash.implicits.sha256Hash
+    import cats.implicits._
+    val paddedSecret = secretTxt.getBytes() ++ Array
+      .fill(secretTxt.getBytes().length - 32)(0.toByte)
```

Comment:
If I understand the intention, shouldn't it be
```suggestion
    val paddedSecret = secretTxt.getBytes() ++ Array
      .fill(32 - secretTxt.getBytes().length)(0.toByte)
```

You can ignore this if I misunderstood the snippet

Reply:
You are right.